controllers/openstackdataplanedeployment_controller.go

```diff
@@ -195,13 +195,21 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 	// All nodeSets successfully fetched.
 	// Mark InputReadyCondition=True
 	instance.Status.Conditions.MarkTrue(condition.InputReadyCondition, condition.ReadyMessage)
+	shouldRequeue := false
+	haveError := false
+
+	globalInventorySecrets := []string{}
+
+	// Gathering individual inventory secrets for later use
+	for _, nodeSet := range nodeSets.Items {
+		// Add inventory secret to list of inventories for global services
+		globalInventorySecrets = append(globalInventorySecrets, fmt.Sprintf("dataplanenodeset-%s", nodeSet.Name))
```

Comment:
not related to your PR, but as a follow up, we should likely factor out the fmt.SPrintf logic into a GetNodeSetInventoryName func in nodeset_types.go, as this same logic is also in inventory.go.

Reply:
also, I don't know that globalInventorySecrets is the right var name here, as it's not global for all NodeSets, just the NodeSets in the Deployment

Reply:
I don't know if naming it `deploymentNodesetsInventorySecrets` would be much better. These are as global as it can get within deployment.